src/App.js

```diff
@@ -1,45 +1,30 @@
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect, useRef } from "react";
```

Comment:
Remove unused imports

Reply:
Also, sort the imports in the following order:

- React official imports (Eg. useState, React)
- Third-party imports (Eg. Modal, Collapse)
- Style sheets

Reply:
The above two apply to all files.